source/includes/examples/tf-example-auth-scram-devtest.rst

```diff
@@ -0,0 +1,28 @@
+.. code-block:: shell 
+   :copyable: true 
+
+   locals {
+     test_user_password = random_password.password.result
+   }
+
+   # Generates 12 characters long random password without special characters
+   resource "random_password" "password" {
+     length  = 12
+     special = false
+   }
+
+   resource "mongodbatlas_database_user" "user1" {
+     username           = var.user[0]
+     password           = local.test_user_password
+     project_id         = var.project_id
+     auth_database_name = "admin"
+     scopes             = var.clusters[0]
+
+     roles {
+       role_name     = "readWriteAny"
+       database_name = var.database_name[0]
+     }
+   }
+
+   output "user1" { value = mongodbatlas_database_user.user1.username }
+   output "userpwd" { value = mongodbatlas_database_user.user1.password }
```

Comment:
this will show an error if sensitive = true is not used: 
```
  18: output "userpwd" { value = mongodbatlas_database_user.user1.password }
│ 
│ To reduce the risk of accidentally exporting sensitive data that was intended to be only internal, Terraform requires that any root
│ module output containing sensitive data be explicitly marked as sensitive, to confirm your intent.
│ 
│ If you do intend to export this data, annotate the output value as sensitive by adding the following argument:
│     sensitive = true
```